solutions/usaco-784.mdx

```diff
@@ -64,4 +64,33 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+```py
+file_in = open('lifeguards.in')
+data = file_in.read().strip().split('\n')
+n = int(data[0])
+
+lifeguards = [list(map(int, data[i + 1].split(' '))) for i in range(n)]
+
+max_time_covered = 0
+
+# Simulate firing each cow and computing the total time covered by the remaining life guards
+# Compute the maximum of these times
+for i in range(n):
+	time_covered = 0
+	for v in range(1, 1001):
```

Comment:
no magic numbers please, maybe a `MAX_M` or something (idk i haven't read the problem) and then replacing `for v in range(1, MAX_M + 1)` would be beeter

Reply:
idk; i think that a constant would help when the number is used multiple times, but it's only used once here so it's not as necessary?

Reply:
It's bronze, so the use of MAX_N could make the code less confusing to read, but either way I don't think it's super important.

Reply:
I'll add it, just in case.